_includes/devdoc/ref_p2p_networking.md

```diff
@@ -693,102 +693,9 @@ d91f4854 ........................... Epoch time: 1414012889
 {% autocrossref %}
 
 *Added in protocol version 311.*
+*Removed in Bitcoin Core 0.14.0*
 
-The `alert` message warns nodes of problems that may affect them or the
```

Comment:
I am not familiar with "internal alerts" mentioned by @btcdrak in https://github.com/bitcoin/bitcoin/pull/7692, is any of this deleted material relevant now?